ramls/examples/inventory-items-and-holdings.json

```diff
@@ -0,0 +1,48 @@
+{
+  "instanceId": "1b74ab75-9f41-4837-8662-a1d99118008d",
+  "itemsAndHoldingsFields": {
+    "items": [
```

Comment:
"itemsAndHoldingsFields" has 3 required fields according to inventory-instance-records.json: instanceId, holdings, items.
This example fails the build because there is no holdings property.

Reply:
I removed redundant level of hierarchy, i.e. mentioned "itemsAndHodlingsFields".